test/util/pod/pod.go

```diff
@@ -123,8 +124,9 @@ func ExecCommand(cs *testclient.ClientSet, pod *corev1.Pod, command ...string) (
 }
 
 // GetLog connects to a pod and fetches log
-func GetLog(cs *testclient.ClientSet, p *corev1.Pod) (string, error) {
-	req := cs.Pods(p.Namespace).GetLogs(p.Name, &corev1.PodLogOptions{})
+func GetLog(cs *testclient.ClientSet, p *corev1.Pod, s time.Duration) (string, error) {
+	sinceSeconds := int64(s.Seconds())
```

Comment:
You can use `logStart := metav1.NewTime(s)` and use the `SinceTime` PodLogOption: `SinceTime: &logStart`

Reply:
Never mind, it's fine this way.

Reply:
Fixed Var name